pkg/test/expectations/expectations.go

```diff
@@ -161,6 +153,7 @@ func ExpectProvisioningCleanedUp(ctx context.Context, c client.Client, controlle
 }
 
 func ExpectProvisioned(ctx context.Context, c client.Client, selectionController *selection.Controller, provisioningController *provisioning.Controller, provisioner *v1alpha5.Provisioner, pods ...*v1.Pod) (result []*v1.Pod) {
+	provisioning.MaxPodsPerBatch = len(pods)
```

Comment:
What's the point of setting this here?

Reply:
This shortcircuits the batcher to only wait for the number of pods we're testing, rather than waiting for a full second per test.